erpnext/payroll/doctype/salary_slip/salary_slip.json

```diff
@@ -625,13 +627,27 @@
    "label": "Leave Details",
    "options": "Salary Slip Leave",
    "read_only": 1
+  },
+  {
+   "fieldname": "gross_year_to_date",
+   "fieldtype": "Currency",
+   "label": "Year To Date",
```

Comment:
Since we already have a field with the label Year To Date, let's rename this to Gross Pay Year To Date? Similarly, Year To Date(Company Currency) can be Gross Pay Year To Date(Company Currency).